gcd/src/main/java/org/spine3/server/storage/datastore/type/DatastoreTypeRegistryFactory.java

```diff
@@ -56,7 +56,7 @@ private DatastoreTypeRegistryFactory() {
     }
 
     /**
-     * Retrieves a default {@link ColumnTypeRegistry ColumnTypeRegistry{@literal <}? extends DatastoreColumnType>}
+     * Retrieves a default {@link ColumnTypeRegistry ColumnTypeRegistry&lt;? extends DatastoreColumnType&gt;}
```

Comment:
The description repeats the signature. 

Instead of having fun with `&lt;` and `&gt;` (and not only here; every time repeating what's written in code), please try to make the aim of the method more clear.